lib/pbench/agent/tool_meister.py

```diff
@@ -898,6 +1016,8 @@ def init_tools(self, data):
         for name, tool_opts in sorted(self._tools.items()):
             try:
                 tklass = self._tool_name_class_mappings[name]
+                if name not in self.persistent_tool_names:
```

Comment:
I'd move this out of the `try` block as the first action taken on the `name`, since we don't need to find a persistent tool class mapping if it is not a persistent tool.